enterprise/frontend/src/embedding-sdk-ee/metabot/index.ts

```diff
@@ -1,5 +1,10 @@
-// Contains side effects that activate the plugin
+// Contains side effects that puts `MetabotQuestion` in the plugin
 import "./MetabotQuestion";
 
+import { METABOT_SDK_EE_PLUGIN } from "embedding-sdk-bundle/components/public/MetabotQuestion/MetabotQuestion";
```

Comment:
Plugins are usually defined in `frontend/src/metabase/plugins` in a single entry point, but this plugin seems to be located within the component. I wonder if we should move the plugin object out of the component at least 🤔

Reply:
I'll write up something, hopefully soon, to explain why I did it in this way https://linear.app/metabase/issue/EMB-1032/documentdiscuss-the-embedding-sdk-plugins